frontend/src/metabase/browse/components/BrowseMetrics.tsx

```diff
@@ -0,0 +1,84 @@
+import { t } from "ttag";
+
+import NoResults from "assets/img/metrics_bot.svg";
+import { useFetchMetrics } from "metabase/common/hooks/use-fetch-metrics";
+import EmptyState from "metabase/components/EmptyState";
+import { DelayedLoadingAndErrorWrapper } from "metabase/components/LoadingAndErrorWrapper/DelayedLoadingAndErrorWrapper";
+import { Box, Flex, Group, Icon, Stack, Text, Title } from "metabase/ui";
+
+import type { MetricResult } from "../types";
+
+import {
+  BrowseContainer,
+  BrowseHeader,
+  BrowseMain,
+  BrowseSection,
+} from "./BrowseContainer.styled";
+import { MetricsTable } from "./MetricsTable";
+
+export function BrowseMetrics() {
+  const metricsResult = useFetchMetrics();
+  const metrics = metricsResult.data?.data as MetricResult[] | undefined;
+
+  const isEmpty = !metricsResult.isLoading && !metrics?.length;
+
+  return (
+    <BrowseContainer>
+      <BrowseHeader role="heading" data-testid="browse-metrics-header">
+        <BrowseSection>
+          <Flex
+            w="100%"
+            h="2.25rem"
+            direction="row"
+            justify="space-between"
+            align="center"
+          >
+            <Title order={1} color="text-dark">
+              <Group spacing="sm">
+                <Icon size={24} color={"var(--mb-color-brand)"} name="metric" />
+                {t`Metrics`}
+              </Group>
+            </Title>
+          </Flex>
+        </BrowseSection>
+      </BrowseHeader>
+      <BrowseMain>
+        <BrowseSection>
+          <Stack mb="lg" spacing="md" w="100%">
+            {isEmpty ? (
+              <MetricsEmptyState />
+            ) : (
+              <DelayedLoadingAndErrorWrapper
+                error={metricsResult.error}
+                loading={metricsResult.isLoading}
+                style={{ flex: 1 }}
+                delay={0}
```

Comment:
What's the point of 0 delay?

Reply:
This is copy pasta code. AFAICT the default is 300ms. I actually prefer that so I removed this prop!

Reply:
Please do the same for models page :pray: 
We need to keep things in sync as much as possible :pray: